lib/typing/ext/parse.go

```diff
@@ -65,8 +62,8 @@ func ParseExtendedDateTime(ctx context.Context, dtString string) (*ExtendedTime,
 	}
 
 	allSupportedDateFormats := supportedDateFormats
-	if len(config.FromContext(ctx).Config.SharedTransferConfig.AdditionalDateFormats) > 0 {
-		allSupportedDateFormats = append(allSupportedDateFormats, config.FromContext(ctx).Config.SharedTransferConfig.AdditionalDateFormats...)
+	if len(additionalDateFormats) > 0 {
```

Comment:
Can `additionalDateFormats` ever be `nil`?

Reply:
Yes, but doesn't matter: https://go.dev/play/p/atKiUvJjpSu